src/quo/components/links/internal_link_card/channel/view.cljs

```diff
@@ -0,0 +1,92 @@
+(ns quo.components.links.internal-link-card.channel.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.channel.style :as status-link-previews-style]
+    [quo.components.links.link-preview.style :as style]
```

Comment:
👎 - This component should have it's own style file. We don't want to put an unexpected dependency on link previews to this component. imo We should just copy paste what is taken from there and have this component have it's own styles. Reusing styles in React should be done via components and encapsulated within components etc 👍